automation/run/run_test.go

```diff
@@ -116,7 +116,7 @@ func TestStateAddTransformEvent(t *testing.T) {
 
 	for i, s := range states {
 		t.Run(fmt.Sprintf("after_event_%d", i), func(t *testing.T) {
-			got := NewState(runID)
+			got := &State{ID: runID}
```

Comment:
Could we keep `NewState` instead of constructing `State` directly? It makes it easier to update all call-sites if another field becomes a required value for construction.

Reply:
That's a great reason! Yes!